index.rst

```diff
@@ -446,148 +374,84 @@ requested time, which will change as the telescope track the object.
 
     await attcs.slew_icrs(ra="20:25:38.85705", dec="-56:44:06.3230", rot_pos=0., target_name="Alf Pav")
 
-As with the :ref:`startup` procedure, this task has not been tested at the telescope yet.
-For now the sanctioned procedure is to execute the slew and track by commanding the pointing
-component individually. This also means the user have to handle the rotator angle
-computations. In this mode we only support setting the rotator position to a certain angle.
-Due to some binding issues we have been trying to keep the rotator as close to zero as possible.
+Note that this task still experience some undesired behaviour, like hanging in until it times out
+or returning before the end of a slew. These issues are mainly due to issues with the `ATMCS`
+either because it still fails to comply with our system architecture (like sending events in
+specific order) or due to issues with the internal control algorithms.
```

Comment:
If you mention there is an issue then you have to talk about what to do when it fails...